README.md

```diff
@@ -83,12 +109,93 @@ writing this
 [.splice()]:
   https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
 
+If you check out the [MDN Array documentation][MDN Arrays], you'll see the full
+list of Array methods that are built into JavaScript on the left. Included in
+the list are the methods above, a number of _iterator methods_ that we'll learn
+about a bit later in this course, and quite a few more. We encourage you to
+explore these methods to get a sense for what you can do with arrays. Below are
+a few that we find particularly helpful.
+
+### Array.prototype.includes()
+
+This method checks whether the value passed as an argument is included in the
+array and returns `true` or `false`:
+
+```js
+const nums = [1, 2, 3];
+nums.includes(2);
+// => true
+```
+
+### Array.prototype.join()
+
+This method allows you to join the elements of an array into a string. If you
+don't pass an argument, the method will separate the elements with commas by
+default:
+
+```js
+const nums = [1, 2, 3];
+nums.join();
+// => "1,2,3"
+```
+
+Optionally, you can pass a separator as an argument; the separator must be a
+string value:
+
+```js
+nums.join(" ");
+// => "1 2 3"
+```
+
+### Array.prototype.reverse()
+
+As you might except, `reverse()` reverses the order of the elements in the
+array:
+
+```js
+const nums = [1, 2, 3];
+nums.reverse();
+// => [3, 2, 1]
+```
+
+**Note**: The `reverse()` method modifies the array _in place_, i.e., it is
```

Comment:
Since `reverse` is destructive, this still replaces the original array (try logging nums in the example below, it'll also be [3,2,1]). It just makes a copy of the reversed array, not the original. 

I don't believe there's a way to preserve the original when you call `reverse` directly on it. You'd have to make a shallow copy of the original, then use reverse on the copy instead, i.e. 

```
const nums = [1, 2, 3];
const numsCopy = [...nums]; 
numsCopy.reverse();
// => [3, 2, 1]
nums;
// => [1, 2, 3]
```

Although since I just realized the spread lesson comes after this one, maybe use `.slice()` to make a copy instead

```
const nums = [1, 2, 3];
const numsCopy = nums.slice(); 
numsCopy.reverse();
// => [3, 2, 1]
nums;
// => [1, 2, 3]
```